tests/compute/test_heterograph.py

```diff
@@ -2503,6 +2503,82 @@ def test_frame_device(idtype):
     assert F.context(ng._node_frames[0]._columns['hh'].storage) == F.ctx()
     assert F.context(ng._edge_frames[0]._columns['h'].storage) == F.cpu()
 
+@parametrize_dtype
+def test_create_block(idtype):
+    block = dgl.create_block(([0, 1, 2], [1, 2, 3]), idtype=idtype, device=F.ctx())
+    assert block.num_src_nodes() == 3
+    assert block.num_dst_nodes() == 4
+    assert block.num_edges() == 3
+
+    block = dgl.create_block(([0, 1, 2], [1, 2, 3]), (4, 5), idtype=idtype, device=F.ctx())
+    assert block.num_src_nodes() == 4
+    assert block.num_dst_nodes() == 5
+    assert block.num_edges() == 3
+
+    sx = F.randn((4, 5))
+    dx = F.randn((5, 6))
+    ex = F.randn((3, 4))
+    block.srcdata['x'] = sx
+    block.dstdata['x'] = dx
+    block.edata['x'] = ex
+
+    g = dgl.block_to_graph(block)
+    assert g.num_src_nodes() == 4
+    assert g.num_dst_nodes() == 5
+    assert g.num_edges() == 3
+    assert g.srcdata['x'] is sx
+    assert g.dstdata['x'] is dx
+    assert g.edata['x'] is ex
```

Comment:
Test for shared storage and features?

Reply:
This is testing shared storage actually.  `is` tests whether the two operands refer to the same object.

Reply:
Got it.